src/content/app/genome-browser/constants/regulationLegends.ts

```diff
@@ -17,30 +17,35 @@
 const regulationLegends = [
   {
     id: 1,
+    stats_api_key: 'promoters',
```

Comment:
I have a slight reservation for this name of the field — "api key" is usually something we use to authenticate with an api. Wouldn't it be clearer if we turned these strings into the value of the `id` field; and moved the current value of the id field to a field called something like `colour_code`:

```
{
    id: 'promoters',
    colour_code: 1,
    ....
}
```